ui/app/src/components/CreateSiteDialog/BlueprintForm.tsx

```diff
@@ -239,7 +239,7 @@ function BlueprintForm(props: BlueprintFormProps) {
                 ? formatMessage(messages.required, { name: formatMessage(messages.siteName) })
                 : inputs.siteNameExist
                 ? formatMessage(messages.nameExist)
-                : ''
+                : formatMessage(messages.fieldMaxLength, { maxLength: siteNameMaxLength })
```

Comment:
Added the helper message in here because the other fields in this dialog have them (for consistency)